hermes-consumers/src/test/java/pl/allegro/tech/hermes/consumers/supervisor/workload/SelectiveWorkBalancerTest.java

```diff
@@ -174,12 +210,23 @@ public void shouldEquallyAssignWorkToConsumers() {
         // given
         List<String> supervisors = ImmutableList.of("c1", "c2");
         List<SubscriptionName> subscriptions = someSubscriptions(50);
-        WorkloadConstraints initialConstraints = WorkloadConstraints.defaultConstraints(2, 200, supervisors.size());
-        SubscriptionAssignmentView currentState = initialState(subscriptions, supervisors, initialConstraints);
+        SelectiveWorkBalancer workBalancer = new SelectiveWorkBalancer();
+        WorkloadConstraints constraints = new WorkloadConstraints(
+                supervisors.size(),
+                2,
+                200,
+                emptyMap(),
+                emptyMap()
+        );
+        SubscriptionAssignmentView currentState = workBalancer.balance(
+                subscriptions,
+                supervisors,
+                new SubscriptionAssignmentView(emptyMap()),
+                constraints
+        ).getAssignmentsView();
```

Comment:
same as in comment above

Reply:
Good point. I used a helper method that already existed in this class.